.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.1
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: cimg/node:16.15.1
+
+set_env: &set_env
```

Comment:
Suggesting to add a simple comment to hint you will use the `set_env` variable which defined some steps options in below steps of jobs.